### ISSUE TITLE
JDK11 MBCS_Tests_CLDR_11 failures

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse-openj9/openj9#16600]
Failure link
------------

From [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_special.functional_x86-64_linux/110/)(`rhel7x64cudart1`):
```
java version "11.0.18" 2023-01-17
IBM Semeru Runtime Certified Edition 11.0.18.0-rc1 (build 11.0.18+10)
Eclipse OpenJ9 VM 11.0.18.0-rc1 (build v0.36.0-release-e68fb241f, JRE 11 Linux amd64-64-Bit Compressed References 20230120_516 (JIT enabled, AOT enabled)
OpenJ9   - e68fb241f
OMR      - f491bbf6f
JCL      - d19b1e858d based on jdk-11.0.18+10)
```
[Rerun in Grinder](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/parambuild/?SDK_RESOURCE=customized&amp;TARGET=MBCS_Tests_CLDR_11_ko_KR_linux_0&amp;TEST_FLAG=&amp;UPSTREAM_TEST_JOB_NAME=&amp;DOCKER_REQUIRED=false&amp;ACTIVE_NODE_TIMEOUT=0&amp;VENDOR_TEST_DIRS=functional&amp;EXTRA_DOCKER_ARGS=&amp;TKG_OWNER_BRANCH=adoptium%3Amaster&amp;OPENJ9_SYSTEMTEST_OWNER_BRANCH=eclipse%3Amaster&amp;PLATFORM=x86-64_linux&amp;GENERATE_JOBS=true&amp;KEEP_REPORTDIR=false&amp;PERSONAL_BUILD=false&amp;DOCKER_REGISTRY_DIR=&amp;ADOPTOPENJDK_REPO=https%3A%2F%2Fgithub.com%2Fadoptium%2Faqa-tests.git&amp;RERUN_ITERATIONS=0&amp;DOCKER_REGISTRY_URL_CREDENTIAL_ID=&amp;LABEL=&amp;EXTRA_OPTIONS=&amp;CUSTOMIZED_SDK_URL=+https%3A%2F%2Fna.artifactory.swg-devops.com%2Fartifactory%2Fsys-rt-generic-local%2Fhyc-runtimes-jenkins.swg-devops.com%2Fbuild-scripts%2Fjobs%2Fjdk11u%2Fjdk11u-linux-x64-openj9-IBM%2F516%2Fibm-semeru-certified-debugimage_x64_linux_11.0.18.0-rc1.tar.gz+https%3A%2F%2Fna.artifactory.swg-devops.com%2Fartifactory%2Fsys-rt-generic-local%2Fhyc-runtimes-jenkins.swg-devops.com%2Fbuild-scripts%2Fjobs%2Fjdk11u%2Fjdk11u-linux-x64-openj9-IBM%2F516%2Fibm-semeru-certified-jre_x64_linux_11.0.18.0-rc1.tar.gz+https%3A%2F%2Fna.artifactory.swg-devops.com%2Fartifactory%2Fsys-rt-generic-local%2Fhyc-runtimes-jenkins.swg-devops.com%2Fbuild-scripts%2Fjobs%2Fjdk11u%2Fjdk11u-linux-x64-openj9-IBM%2F516%2Fibm-semeru-certified-jdk_x64_linux_11.0.18.0-rc1.tar.gz+https%3A%2F%2Fna.artifactory.swg-devops.com%2Fartifactory%2Fsys-rt-generic-local%2Fhyc-runtimes-jenkins.swg-devops.com%2Fbuild-scripts%2Fjobs%2Fjdk11u%2Fjdk11u-linux-x64-openj9-IBM%2F516%2Fibm-semeru-certified-testimage_x64_linux_11.0.18.0-rc1.tar.gz&amp;BUILD_IDENTIFIER=&amp;ADOPTOPENJDK_BRANCH=master&amp;LIGHT_WEIGHT_CHECKOUT=false&amp;USE_JRE=false&amp;ARTIFACTORY_SERVER=na.artifactory.swg-devops&amp;KEEP_WORKSPACE=false&amp;USER_CREDENTIALS_ID=83181e25-eea4-4f55-8b3e-e79615733226&amp;JDK_VERSION=11&amp;DOCKER_REGISTRY_URL=&amp;ITERATIONS=1&amp;VENDOR_TEST_REPOS=git%40github.ibm.com%3Aruntimes%2Ftest.git&amp;JDK_REPO=git%40github.com%3Aibmruntimes%2Fopenj9-openjdk-jdk11&amp;JCK_GIT_BRANCH=main&amp;RELEASE_TAG=&amp;OPENJ9_BRANCH=master&amp;OPENJ9_SHA=&amp;JCK_GIT_REPO=&amp;VENDOR_TEST_BRANCHES=master&amp;OPENJ9_REPO=https%3A%2F%2Fgithub.com%2Feclipse-openj9%2Fopenj9.git&amp;UPSTREAM_JOB_NAME=&amp;CLOUD_PROVIDER=&amp;CUSTOM_TARGET=&amp;VENDOR_TEST_SHAS=&amp;JDK_BRANCH=openj9&amp;LABEL_ADDITION=&amp;ARTIFACTORY_REPO=&amp;ARTIFACTORY_ROOT_DIR=&amp;UPSTREAM_TEST_JOB_NUMBER=&amp;DOCKERIMAGE_TAG=&amp;JDK_IMPL=openj9&amp;TEST_TIME=&amp;SSH_AGENT_CREDENTIAL=83181e25-eea4-4f55-8b3e-e79615733226&amp;AUTO_DETECT=true&amp;SLACK_CHANNEL=%23rt-jenkins&amp;DYNAMIC_COMPILE=true&amp;ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH=adoptium%3Amaster&amp;APPLICATION_OPTIONS=&amp;CUSTOMIZED_SDK_URL_CREDENTIAL_ID=4e18ffe7-b1b1-4272-9979-99769b68bcc2&amp;ARCHIVE_TEST_RESULTS=false&amp;NUM_MACHINES=&amp;OPENJDK_SHA=&amp;TRSS_URL=http%3A%2F%2Ftrss1.fyre.ibm.com&amp;USE_TESTENV_PROPERTIES=false&amp;BUILD_LIST=functional&amp;UPSTREAM_JOB_NUMBER=&amp;STF_OWNER_BRANCH=adoptium%3Amaster&amp;TIME_LIMIT=20&amp;JVM_OPTIONS=&amp;PARALLEL=None) - Change TARGET to run only the failed test targets.

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------
```
[2023-01-20T05:19:04.647Z] variation: NoOptions
[2023-01-20T05:19:04.647Z] JVM_OPTIONS:  

[2023-01-20T05:19:13.929Z] not ok 17 - diff CurrencyTest-ko-KR-DEFAULT.log expected_CurrencyTest-ko-KR-DEFAULT.log
[2023-01-20T05:19:13.929Z] #   Failed test 'diff CurrencyTest-ko-KR-DEFAULT.log expected_CurrencyTest-ko-KR-DEFAULT.log'
[2023-01-20T05:19:13.929Z] #   at /home/jenkins/workspace/Test_openjdk11_j9_special.functional_x86-64_linux/aqa-tests/TKG/../../jvmtest/functional/MBCS_Tests/CLDR_11/tap_compare.pl line 57.
[2023-01-20T05:19:13.929Z] not ok 18 - diff CurrencyTest-ko-KR-CLDR.log expected_CurrencyTest-ko-KR-CLDR.log
[2023-01-20T05:19:13.929Z] #   Failed test 'diff CurrencyTest-ko-KR-CLDR.log expected_CurrencyTest-ko-KR-CLDR.log'
[2023-01-20T05:19:13.929Z] #   at /home/jenkins/workspace/Test_openjdk11_j9_special.functional_x86-64_linux/aqa-tests/TKG/../../jvmtest/functional/MBCS_Tests/CLDR_11/tap_compare.pl line 57.
[2023-01-20T05:19:13.929Z] not ok 19 - diff CurrencyTest-ko-KR-JRE.log expected_CurrencyTest-ko-KR-JRE.log
[2023-01-20T05:19:13.929Z] #   Failed test 'diff CurrencyTest-ko-KR-JRE.log expected_CurrencyTest-ko-KR-JRE.log'
[2023-01-20T05:19:13.930Z] #   at /home/jenkins/workspace/Test_openjdk11_j9_special.functional_x86-64_linux/aqa-tests/TKG/../../jvmtest/functional/MBCS_Tests/CLDR_11/tap_compare.pl line 57.
[2023-01-20T05:19:13.930Z] not ok 20 - diff CurrencyTest-ko-KR-SPI.log expected_CurrencyTest-ko-KR-SPI.log
[2023-01-20T05:19:13.930Z] #   Failed test 'diff CurrencyTest-ko-KR-SPI.log expected_CurrencyTest-ko-KR-SPI.log'
[2023-01-20T05:19:13.930Z] #   at /home/jenkins/workspace/Test_openjdk11_j9_special.functional_x86-64_linux/aqa-tests/TKG/../../jvmtest/functional/MBCS_Tests/CLDR_11/tap_compare.pl line 57.

[2023-01-20T05:19:13.930Z] # Looks like you failed 4 tests of 35.
[2023-01-20T05:19:13.930Z] -----------------------------------
[2023-01-20T05:19:13.930Z] MBCS_Tests_CLDR_11_ko_KR_linux_0_FAILED
```

Also failed at [JDK11 0.36 s390x](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_special.functional_s390x_linux/107/)(`ub18s390xrt-1-8`)
[JDK11 0.36 aarch64](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_special.functional_aarch64_linux/82/)(`ub18-aarch64-4`)

A similar issue was reported at 
* https://github.com/adoptium/aqa-tests/issues/4271

**Comment by smlambert** [comment]
Fix is merged into master via https://github.com/adoptium/aqa-tests/pull/4272 (still needs to go into the v0.9.6-release branch).

**Comment by pshipton** [comment]
I assume this was resolved as failures in jdk11 0.36.1 rc1 were not reported.
